src/store/activities/actions.js

```diff
@@ -7,17 +7,21 @@ export const addActivity = activity => ({
 });
 
 export const startAddActivity = (activity = {}) => dispatch => {
-  return database
-    .ref('activities/items')
-    .push(activity)
-    .then(ref => {
-      dispatch(
-        addActivity({
-          id: ref.key,
-          ...activity,
-        }),
-      );
-    });
+  try {
```

Comment:
`try catch` block is proper, except that you are doing nothing for user :P Think about it in short future, how to prompt users about errors in nice way :) 
I wanted you to use `async/await` for practice, but you still uses `Promises` :) So, I would leave `changes request` 😱